modules/standard/Map.chpl

```diff
@@ -254,6 +267,19 @@ module Map {
       return result;
     }
 
+/*
+    pragma "no doc"
```

Comment:
Either remove this code or provide a comment illustrating why it doesn't work, for now? (Perhaps linking back to the cast to ref issue #14937).